### DISCUSSION TITLE
[Suggestion] Make your score double if you get to a multiple of 10 in endless with all three lives intact

### DISCUSSION THREAD

**Discussion by Bulba413** [chrislo27/PolyrhythmMania#54] [Ideas]
The old Game & Watches had this mechanic where if you reached a point where you would normally regain a life, but hadn't missed yet, all score gained from that point onward until you lost a life would be doubled. I think this could be a fun mechanic to implement in this game's endless mode as a reward for playing well (though it wouldn't be available in daredevil mode for obvious reasons)

**Comment by chrislo27** [comment]
I think this is an interesting suggestion! It would certainly make the Daily Challenge leaderboard more interesting. Would be interested to see what others think about this.
